modules/tinymce/src/themes/silver/main/ts/ui/context/ContextFormTextInput.ts

```diff
@@ -17,9 +17,16 @@ export const renderContextFormTextInput = (
     components: [ GuiFactory.text(label) ]
   }));
 
+  const placeholder = ctx.placeholder.fold( Fun.constant({}), (p) => ({ placeholder: providers.translate(p) }));
```

Comment:
Just for future reference
```suggestion
  const placeholder = ctx.placeholder.map((p) => ({ placeholder: providers.translate(p) })).getOr({});
```

Reply:
One could argue that both these approaches are fine but the suggested one doesn't need Fun to be imported also I think we use the suggested pattern more often.

Reply:
The original avoids creating an extra wrapping `Optional` instance from the `map` result, but it adds a throwaway function.

We use the `map().getOr()` pattern often because it's more concise, at least IMO. The only reason we avoid it is if calculating the fallback is expensive; that's when we either go with `fold` or perhaps `.map().getOrThunk()`.